jwst/transforms/tags/jwst_models.py

```diff
@@ -248,8 +323,14 @@ class SnellType(TransformType):
 
     @classmethod
     def from_tree_transform(cls, node, ctx):
-        return Snell(node['prism_angle'], node['kcoef'], node['lcoef'], node['tcoef'],
-                     node['ref_temp'], node['ref_pressure'], node['temp'], node['pressure'])
+        return Snell(node['prism_angle'],
+                     node['kcoef'],
+                     node['lcoef'],
+                     node['tcoef'],
+                     node['ref_temp'],
+                     node['ref_pressure'],
+                     node['temp'],
+                     node['pressure'])
```

Comment:
Please put this back in the previous formatting.
I don't want to be a pain but accepting this means pain in the future because we have different settings and will have to constantly resolve conflicts. A better solution may be to have the grism stuff in a separate module. I'll leave it up to you but there's really no need to change classes that do not belong to this PR.

Reply:
sorry  - I thought I had reformatted this one too, will do.